Samples/PublicSamples/RecordingBot/src/RecordingBot.Services/Http/Controllers/PlatformCallController.cs

```diff
@@ -65,8 +67,9 @@ public PlatformCallController(IGraphLogger logger, IBotService botService)
 
             RequestValidationResult result;
 
-                var httpRequestMessage = new System.Net.Http.HttpRequestMessage();
-                httpRequestMessage.Headers.Authorization = new System.Net.Http.Headers.AuthenticationHeaderValue("Bearer", Request.Headers.Authorization);
+                var httpRequestMessage = new HttpRequestMessage();
+                httpRequestMessage.Headers.Authorization = new AuthenticationHeaderValue("Bearer", Request.Headers.Authorization);
```

Comment:
Request.Headers.Authorization needs to be splitted from Bearer( see how it is done in main Branch)